tests/unit/CssState.js

```diff
@@ -22,9 +22,9 @@ define([
 			});
 			widget.deliver();
 
-			assert($(widget).hasClass("d-error"), "error state");
-			assert($(widget).hasClass("d-disabled"), "disabled");
-			assert($(widget).hasClass("d-checked"), "checked");
+			assert(widget.hasClass("d-error"), "error state");
+			assert(widget.hasClass("d-disabled"), "disabled");
+			assert(widget.hasClass("d-checked"), "checked");
```

Comment:
It's kind-of bad form for the test files to use the widget methods, makes the tests less "objective" (can't think of a better word right now).   How about just using widget.classList.has()?   Likewise for the other test files.